docs/build/build-hrmp-channels.md

```diff
@@ -30,17 +30,26 @@ the Relay Chain.
 
 ## Opening HRMP Channels with System Parachains
 
-Opening an HRMP channel with a system parachain requires a referendum. Like all other governance
-proposals, proposers should follow best practices like opening a discussion on
+Opening an HRMP channel with a system parachain requires an
+[OpenGov referendum](../learn/learn-guides-polkadot-opengov.md) using the
+[General Admin Track](../learn/learn-polkadot-opengov-origins.md#general-admin). Like all other
+governance proposals, proposers should follow best practices like opening a discussion on
 [Polkassembly](https://polkadot.polkassembly.io/) or [Subsquare](https://polkadot.subsquare.io/) and
 then submitting the proposal on-chain.
 
+Before submitting the proposal, both the system parachain's and the parachain's sovereign accounts
+(both [system accounts](../learn/learn-account-advanced.md#system-accounts)) must have the channel
+deposit of 20 DOT plus the
+[existential deposit](../learn/learn-accounts.md#existential-deposit-and-reaping) and some extra
+funds to cover for fees. Anyone can send those funds via `balances.transferKeepAlive` call to the
+system parachain's sovereign account (if funds are not already available). System parachains have ID
```

Comment:
Why the complexity? This can simply say, "at the time of execution, each parachain's sovereign account must have a free balance greater than or equal to the sum of the sender deposit, recipient deposit, and existential deposit."

Reply:
implemented